app/views/two_factor_authentications/_activate.haml

```diff
@@ -0,0 +1,10 @@
+.col-md-12
+  .row
+    .col-md-12
+      %h3= t("two_factor_auth.title")
+
```

Comment:
I think a small informational sentence here would help novice users understand what this is about. It can probably be kept on other pages too. Something like *"Two factor authentication is a powerful way to ensure you are the only one able to login to your account. It requires you to enter a 6 digits code generated from your phone after you entered your password to prove to diaspora\* that you are the owner. Be careful though: if you lose your phone app **and** the recovery codes created when you activate this feature, access to your diaspora\* account will be blocked forever."*

@goobertron can you please check the wording?

Reply:
Hi, sorry for the delay. For some reason I didn't receive (or, more likely, didn't notice) a ping for this.

I'm not sure I fully understand how 2FA works. So I don't want to edit without understanding it. Your text suggests that this is just for phones, and requires an app. Is that correct? Once I know, I'll try to suggest text.

Reply:
If you don't get it with my text above, it's a proof that it's not good enough :stuck_out_tongue: 
Two factors authentication requires the user to enter a code provided by an application when he logs in. The application needs to be set up first, and that's done in the user settings. During that process, recovery codes are generated. They allow the user to log in without the code provided by the app (mainly used if the access to the app is lost). Is that clearer?

Reply:
Thanks. 

@Flaburgan, I don't feel competent right now to decide whether the text is factually correct, so I'll just make a few grammatical/stylistic corrections and leave it to someone who knows more about this to approve the content of the text.

> Two-factor authentication is a powerful way to ensure you are the only one able to sign in to your account. When signing in, you will enter a 6-digit code along with your password to prove your identity. Be careful though: if you lose your phone and the recovery codes created when you activate this feature, access to your diaspora* account will be blocked forever.

As this appears to be about how to sign in with 2FA rather than how to set it up, I have deleted a couple of parts that read as if they concern setting up 2FA. Also, you say 'if you lose your phone app'. As I understand it, this concerns signing in to the diaspora\* web-app itself, rather than any third-party app that a user might have on their phone in order to access the diaspora\* web-app. So I deleted 'app' from that clause in order to avoid that confusion.

I hope this is helpful, and I hope that someone else can confirm whether the facts in the text are correct and clear.

Reply:
It's shorter and the meaning is still there, so :+1: 

@lislis will ultimately choose anyway :stuck_out_tongue:

Reply:
Reads well to me :)